pype/plugins/global/publish/submit_publish_job.py

```diff
@@ -21,8 +21,8 @@ def _get_script():
         module_path = module_path[: -len(".pyc")] + ".py"
 
     module_path = os.path.normpath(module_path)
-    mount_root = os.path.normpath(os.environ["PYPE_STUDIO_CORE_MOUNT"])
-    network_root = os.path.normpath(os.environ["PYPE_STUDIO_CORE_PATH"])
+    mount_root = os.path.normpath(os.environ["PYPE_CORE_MOUNT"])
+    network_root = os.path.normpath(os.environ["PYPE_CORE_PATH"])
```

Comment:
while we're doing this. I would personally change this from `PYPE_CORE_PATH` to `PYPE_CORE_UNC` to make it more clear.

Reply:
That was actually my forgotten question, should we use `path` or `unc` key for network paths in roots.json?

Reply:
just to remind you that UNC is strictly Windows concept.

Reply:
true that. maybe `PYPE_CORE_NETWORK`?

to be honest on linux this almost always identical to `PYPE_CORE_MOUNT` anyway, so it's questionable what to call it.